resources/puppetlabs/lein-ezbake/template/foss/ext/debian/ezbake.service.erb

```diff
@@ -12,6 +12,9 @@ Restart=on-failure
 StartLimitBurst=5
 #set default privileges to -rw-r-----
 UMask=027
+<% if EZBake::Config[:open_file_limit] -%>
```

Comment:
So because the value is wrapped in '' in ezbake.rb, it will *always* be non-nil/truthy, even if not set intentionally. This breaks anything not specifying the open_file_limit.